### PR TITLE
Address Lighthouse Reports: [a11y, seo, best practices]

### PR DESCRIPTION
**Pull request by nisarhassan12**
Addresses parts of #135 

In this PR I have addressed most of the things reported by Lighthouse relating to accessibility, SEO and best practices (there are still some things remaining to be addressed relevant to these 3 I have opened issues to keep track of them).  
For performance-related stuff, I'll work on other PRs just like this one https://github.com/gitpod-io/website/pull/160.

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/804"></a>

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/804"></a>

### CODE REVIEW

**Comment by nisarhassan12** [comment]
@ChristinFrohne I ran the audits for all the pages of the site apart from the older blog posts that were like more than 8 months old. 

I noticed some small issues on a few pages listed here https://www.gitpod.io/docs/languages-and-frameworks which I have ignored because they were really minor. I could look into those if you say they are not straightforward to fix.

**Comment by nisarhassan12** [comment]
Thanks, @ChristinFrohne I just want you to run LightHouse on a few pages to see the improvements yourself otherwise I'm sure of the changes and improvements made here. 

There are few text changes in this PR. It would be nice if you go over them and let me know if they look good to you. You can view the text changes view going through https://github.com/gitpod-io/website/pull/804/files 
image

File: src/contents/pricing.ts [hunk 1]
```diff
@@ -72,7 +72,7 @@ export const otherPlans = [
       "Gitpod self-hosted is the best solution for teams who want to keep full data control or use Gitpod in private networks.",
       "Install Gitpod Self-Hosted on <strong>Google Cloud Platform</strong> and <strong>K3s</strong>.",
     ],
-    btnText: "Learn more",
+    btnText: "Learn More about Self Hosted",
```
Comment (ChristinFrohne) [inline src/contents/pricing.ts:75]: Let's shorten it to "More about Self-Hosted"

File: src/routes/docs/languages/javascript.md [hunk 2]
```diff
@@ -39,7 +39,7 @@ tasks:
 
 <span aria-hidden="true">👆</span> In the above example, we are telling Gitpod to run what is in the `init` phase at the time of workspace initialization and then afterwards run whatever is in the `command` phase.
 
-You can read more about start tasks [here](/docs/config-start-tasks).
+You can read more about start tasks [on this page](/docs/config-start-tasks).
```
Comment (ChristinFrohne) [inline src/routes/docs/languages/javascript.md:42]: Shorten it to "Read more about **start tasks**"

**Review by ChristinFrohne — CHANGES_REQUESTED** [review]

File: src/contents/home/index.ts [hunk 3]
```diff
@@ -106,7 +106,7 @@ export const projects: Project[] = [
   },
   {
     logo: "svg/projects/go.svg",
-    title: "Go",
+    title: "Go Programming Language",
```
[the remark this answers is not in the document]
Comment (nisarhassan12) [reply src/contents/home/index.ts:109]: We can use the word Golang as is the case for the title of its website https://golang.org/ 
image

File: src/routes/docs/quickstart/go.md [hunk 4]
```diff
@@ -9,7 +9,7 @@ title: Go template
 
 # Go Quickstart
 
-Learn how to set-up and understand the benefits of Gitpod **in less than 5 min** with our [Go](https://github.com/gitpod-io/template-golang-cli) template.
+Learn how to set-up and understand the benefits of Gitpod **in less than 5 min** with our [Go Programming Language](https://github.com/gitpod-io/template-golang-cli) template.
```
[the remark this answers is not in the document]
Comment (nisarhassan12) [reply src/routes/docs/quickstart/go.md:12]: https://github.com/gitpod-io/website/pull/804#discussion_r689657081

**Pull request merged**